### ISSUE TITLE
No English Language

### ISSUE THREAD

**Issue by monowar1993** [huanghaibin-dev/CalendarView#70]
Hello,
Its really a great library with lots of customization. Is there any way to have support for English Language? All strings used in this library is in Chinese. Is there any way to get them on English?

**Comment by huanghaibin-dev** [comment]
I'm sorry, all the comments in the library are in chinese. All strings come from festivals. I do my best to support English in the next version.  my english is so poor

**Comment by monowar1993** [comment]
Comments are okay. But lunar day names, months names should be in English. You can use android localization to create separate string,xml file for English and Chinese. 
I also have a question. Can I add multiple scheme in a single date?

**Comment by huanghaibin-dev** [comment]
Of course, You need to use custom MonthView and WeekView.  I suggest putting information into scheme,  such as scheme= "info1_info2_info3",  and then drawing to canvas based on different information. Thank you for your advice.

**Comment by monowar1993** [comment]
Can I add multiple scheme by using 
schemes.add(getSchemeCalendar(year, month, 25, 0xFFedc56d, "Football game"));
schemes.add(getSchemeCalendar(year, month, 25, 0xFF13acf0, "Dinner at night"));

I want them separate.

**Comment by huanghaibin-dev** [comment]
Sorry, the current version is not supported multiple scheme color, I will continue to improve.

**Comment by monowar1993** [comment]
It is very necessary. Thanks for making such a cool library. Please improve by adding English strings and multiple scheme support for single date.

**Comment by huanghaibin-dev** [comment]
Hello, the new version has been released. Thank you for your support.

**Comment by monowar1993** [comment]
thanks for the update. Also to customize there should be option to add from xml. That will give more flexibility for UI design.

**Comment by huanghaibin-dev** [comment]
It's not good for memory by using  XML layout,I don't know how to express,Using XML Layout will increase at least 3*42 Views. if use lunar calendar, it will increase 402 Views at least.

**Comment by monowar1993** [comment]
![calendar](https://user-images.githubusercontent.com/13250644/35931318-865e9112-0c5e-11e8-91bb-63f961dcfcbc.jpg)

How Can I achieve a cell like that. White circle with three dots. Those three dots represents 3 different types of reminder. Can you help me regrading this. If you can provide a example in your demo. That would be great mate.

**Comment by huanghaibin-dev** [comment]
@monowar1993 I've done it. Please look at the solar package.

**Comment by monowar1993** [comment]
Thats really great. Thanks mate.

**Comment by monowar1993** [comment]
Hello mate. I found an issue. I am developing a app with lots of schemes. So the schemes data is growing all the time as I swipe from months to months. Cause on monthSelected I am adding data to schemes. When I swipe too far and too fast 
java.util.concurrentmodificationexception occurs. Please look into this issue.

**Comment by huanghaibin-dev** [comment]
@monowar1993 When you using foreach, don't add scheme in other thread, You can stick out your key code

**Comment by monowar1993** [comment]
So you are saying I'll need to add schemes on Main thread?

**Comment by huanghaibin-dev** [comment]
Yes，when ArrayList use iterator，don.t modify it, this operation is wrong
